### PR TITLE
Give the projects page some love

### PR DESCRIPTION
**Pull request by Robbe7730**
Add new projects and give them an arbitrary priority.

### CODE REVIEW

**Comment by Robbe7730** [comment]
I may look into putting projects into columns, so we don't have the long lines of text...

**Comment by Robbe7730** [comment]
Ping

File: content/projects/haldis.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ title: Haldis
 github: https://github.com/ZeusWPI/Haldis
 site: https://zeus.ugent.be/haldis
 logo_image: Logos_Haldis.svg
+priority: 35
 ---
 
-Haldis is your friendly neighbourhood servant. He exists so lazy fucks like you and me don't need to keep tabs of who is ordering what from where. Start an order and let people add items with a simple mouse-click! No more calculating prices and making lists! Be lazier today! [http://zeus.ugent.be/haldis](http://zeus.ugent.be/haldis)
+Haldis is your friendly neighbourhood servant. He exists so lazy fucks like you and me don't need to keep tabs of who is ordering what from where. Start an order and let people add items with a simple mouse-click! No more calculating prices and making lists! Be lazier today! [https://zeus.gent/haldis](https://zeus.gent/haldis)
```
Comment (Jan-PieterBaert) [inline content/projects/haldis.md]: the link should be https://zeus.ugent.be/haldis or https://haldis.zeus.gent

File: content/projects/tab.md [hunk 2]
```diff
@@ -3,6 +3,7 @@ title: Tab
 github: https://github.com/ZeusWPI/Tab
 site: https://zeus.ugent.be/tab/
 logo_image: Logos_Tab.svg
+priority: 40
 ---
 
-💰 Yes. We have to drink. But we also have to pay. This does the paying part. [https://zeus.ugent.be/tab](https://zeus.ugent.be/tab)
+💰 Yes. We have to drink. But we also have to pay. This does the paying part. [https://zeus.gent/tab](https://zeus.gent/tab)
```
Comment (Jan-PieterBaert) [inline content/projects/tab.md]: the link should be https://zeus.ugent.be/tab or https://tab.zeus.gent

File: content/projects/tap.md [hunk 3]
```diff
@@ -3,6 +3,7 @@ title: Tap
 github: https://github.com/ZeusWPI/Tap
 site: https://zeus.ugent.be/tap/
 logo_image: Logos_Tap.svg
+priority: 40
 ---
 
-💰 Yes. We have to drink. But we also have to pay. This does the drinking part. [https://zeus.ugent.be/tap](https://zeus.ugent.be/tap)
+💰 Yes. We have to drink. But we also have to pay. This does the drinking part. [https://zeus.gent/tap](https://zeus.gent/tap)
```
Comment (Jan-PieterBaert) [inline content/projects/tap.md]: the link should be https://zeus.ugent.be/tap or https://tap.zeus.gent

**Pull request merged**